dirs/dirs.go

```diff
@@ -224,6 +225,7 @@ func SetRootDir(rootdir string) {
 	SystemApparmorCacheDir = filepath.Join(rootdir, "/etc/apparmor.d/cache")
 
 	CloudMetaDataFile = filepath.Join(rootdir, "/var/lib/cloud/seed/nocloud-net/meta-data")
```

Comment:
what uses this? (grep says nothing) is it related to this work?

Reply:
that's probably a question for @mvo5, I know image does some cloud-init related stuff, but I don't know if this one is 15.04 leftover

Reply:
tbh that's probably fine then :-)

Reply:
This looks very much like a leftover from the old days. Fwiw, `git blame` blames @chipaca for touching it last (8759e4074e92386d33851278aff3fd31cbab27f9 from 2015) - the commit message is a bit obscure though.